cupy/_creation/from_data.py

```diff
@@ -107,6 +107,22 @@ def ascontiguousarray(a, dtype=None):
     return _core.ascontiguousarray(a, dtype)
 
 
+def asfarray(a, dtype=None):
```

Comment:
Could you update the implementation so that it matches with the behavior of NumPy?

* The default dtype is `numpy.float_`
* If inexact dtype is specified, ignore it

https://github.com/numpy/numpy/blob/v1.21.0/numpy/lib/type_check.py#L84-L114

Reply:
Please implement this function at: https://github.com/cupy/cupy/blob/v10.0.0rc1/cupy/_manipulation/kind.py#L5

As this function is "Array manipulation routines > Changing kind of array" category in NumPy reference: https://numpy.org/doc/stable/reference/routines.array-manipulation.html#changing-kind-of-array